temporian/core/event_set_ops.py

```diff
@@ -4436,3 +4437,115 @@ def assign(
         from temporian.core.operators.glue import assign
 
         return assign(self, **others)
+
+    def before(
+        self: EventSetOrNode,
+        timestamp: Union[int, float, datetime],
+    ) -> EventSetOrNode:
+        """Filter events [`EventSet`][temporian.EventSet] that happened before a
```

Comment:
I am not a native speaker, but I think "filter events before A" might be understood as removing the ones before A (i.e. the opposite of this operator). I wonder if this could be made less ambiguous.

Reply:
I would understand "filter out..." as the opposite, since I'm not a native speaker either, run it through chatGPT the question:
```
Given the following documentation of a function called f:

"Filters events that happened before a particular timestamp."

What do you understand that the function should do?
```
And it understood the correct thing:
```
def f(events, timestamp):
    # Filter events that happened before the specified timestamp
    filtered_events = [event for event in events if event['timestamp'] < timestamp]
    return filtered_events
```